### PR TITLE
Train - editorial pass

### PR DESCRIPTION
**Pull request by aaronmarkham**

### CODE REVIEW

File: train.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ valid_data = gluon.data.DataLoader(
 
 We reimplement the same LeNet introduced before. One difference here is that we changed the weight initialization method to `Xavier`, which is a popular choice for deep convolutional neural networks.
```
Comment (aaronmarkham) [inline train.md]: Any comment on why, other than it is popular? Would it be worth mentioning the range of init methods that can be used?

File: train.md [hunk 2]
```diff
@@ -91,26 +91,26 @@ with net.name_scope():
...
 trainer = gluon.Trainer(net.collect_params(),
                         'sgd', {'learning_rate': 0.1})
 ```
 
-The `trainer` is created with all parameters (both weights and gradients) in `net`, later on, we only need to call the `step` method to update its weights.
+The `trainer` is created with all parameters (both weights and gradients) in `net`. Later on, we only need to call the `step` method to update its weights.
 
 ## Train
 
 We create two auxiliary functions before training. Even though both are supported by built-in functions in Gluon, we found it's straightforward to implement them from scratch. One calculates the model accuracy, the other transforms the data into `(batch, channel, height, weight)` format and normalize its values.
```
Comment (aaronmarkham) [inline train.md]: Again, why? Built-in is nice, and simplifies the code. What benefit are we getting by doing it from scratch? As a learning opportunity, you could then show the one-liner or the easier built-in code...

File: train.md [hunk 3]
```diff
@@ -156,7 +156,7 @@ Time %.1f sec" % (
 
 ## Save the model
 
-Finally, we save the trained parameters into disk so that we can use them later.
+Finally, we save the trained parameters onto disk, so that we can use them later.
```
Comment (aaronmarkham) [inline train.md]: I hope we'll dig into this more later. Maybe cross link to where loading is covered?

File: train.md [hunk 4]
```diff
@@ -14,19 +14,19 @@ from time import time
 
 ## Get data
 
-The handwritten digit MNIST dataset is one of the most commonly used datasets in deep learning. But it is too simple to get a 99% accuracy. Here we use a similar but slightly more complicate dataset called FashionMNIST. The goal is no longer to classify number, but clothing.
+The handwritten digit MNIST dataset is one of the most commonly used datasets in deep learning. But it is too simple to get a 99% accuracy. Here we use a similar but slightly more complicated dataset called FashionMNIST. The goal is no longer to classify numbers, but clothing types instead.
```
Comment (aaronmarkham) [inline train.md]: Is one outcome here to show something that achieves 99% accuracy then? Or show some variety?

Comment (mli) [reply train.md]: just want to use a slightly different dataset to replace the over-used mnist
